robottelo/helpers.py

```diff
@@ -253,3 +253,20 @@ def remove_katello_ca(hostname=None):
             )
             if result.return_code != 0:
                 raise AssertionError('Failed to reset the rhsm.conf')
+
+
+def add_remote_execution_ssh_key(hostname, key_path=None, **kwargs):
```

Comment:
@oshtaier added `robottelo.helpers.add_remote_execution_ssh_key('123455.redhat,com')` so this helpers does both things 1 ) get key from sat machine 2) add to the client machine.

I am not sure on how the machine running the test will connect to the client box, is it using username and password? or some default key?

BTW: that **kwargs passes username, password, key to the client connection